aggregator/options.go

```diff
@@ -612,7 +656,9 @@ func (o *options) computeFullGaugePrefix() {
 
 // By default we use e.g. ".p50", ".p95", ".p99" for the 50th/95th/99th percentile.
 func defaultTimerQuantileSuffixFn(quantile float64) []byte {
-	return []byte(fmt.Sprintf(".p%0.0f", quantile*100))
+	s := fmt.Sprintf(".p%0.0f", quantile*10000)
```

Comment:
It's not easy to find the integer value for a float, see: https://play.golang.org/p/SKBCfflTwD, also if we want to keep multiplying until we reach a "integer" we would need to assume a magic epsilon to check if a float is integer or not anyway, so I'm just gonna assume the precision instead. Will change the trimming to be more dynamic.